apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/Emulator/metadata_server.py

```diff
@@ -664,6 +770,22 @@ def __init__(self, items, lock):
     assert isinstance(items, list)
     MetadataDirectory.__init__(self, items, lock)
 
+  def deleteChild(self, child_name):
+    """Attempts to delete the child with the specified name and returns whether the operation was successful.
+
+    Args:
+      child_name: The name where the value should be deleted.
+    """
+    try:
+      index = int(child_name)
```

Comment:
just on principle, keep the body of the try clause as tight as feasible: here, just insert an `else:` after line 780